### DISCUSSION TITLE
paperPlane-纸飞机

### DISCUSSION THREAD

**Discussion by 404ll** [creatorsand/.github#126] [Ideas]
## 一句话 IDEA：
paperPlane-纸飞机：将想法写在纸飞机上，存储在walrus上，匿名的发送至链上。

## 我的角色：

项目发起人&合约&前端&UI

### 其他角色需求（不包含发起人）：
前端

## 团队成员
（社群昵称）。：负责完善前端

## 社群昵称：
Elemen

## 个人简介：

正在学习的小白，努力成为厉害的开发者

## 产品介绍
类似微信漂流瓶，利用walrus将纸飞机的内容存储在链上，避免因平台问题产生的数据丢失等。

### 产品是给谁用的？
目标群体为有情感倾诉需求的个人用户

### 其他产品有什么痛点，用户为什么要选择你的产品？
传统的”漂流瓶“类项目无法保证用户信息的安全性以及隐私性，本项目采用walrus分布式存储协议将纸飞机中的文字内容和图片内容上链，以免数据丢失。

### ppt
https://www.canva.cn/design/DAGZv81TxsU/oVDrZBG6Zcjv6b5h-oLJ2A/edit?ui=eyJEIjp7IlAiOnsiQiI6ZmFsc2V9fX0
### 仓库地址
https://github.com/404ll/ParperPlane

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
